sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -68,10 +74,28 @@ def main(config:str,
         Output verbosity. 0:Error, 1:Warning, 2:Info(default), 3:Debug
     overwrite : bool
         if False, do not re-check existing entries
+    logger : logging
+        When to output the print statements
+
+
     """
+
```

Comment:
The function arg logger now conflicts with the global "logger" variable.  We will need to investigate later whether all of this is really necessary ... but for now please fix it like this:
```
if logger is None:
    logger = globals()['logger']
else:
    globals()['logger'] = logger
```

Reply:
I'll fix and run a test on the site books. I hope I survive contact with data...

Reply:
We're good now.